apps/storybook-react/package.json

```diff
@@ -11,6 +11,7 @@
   },
   "devDependencies": {
     "@chromatic-com/storybook": "^1.9.0",
+    "@metamask/design-system-react": "workspace:^",
```

Comment:
As we are using design-system-react components it's now a dev dependency of storybook-react